eth/api.go

```diff
@@ -1376,6 +1376,140 @@ func (api *PublicDebugAPI) SeedHash(number uint64) (string, error) {
 	return fmt.Sprintf("0x%x", hash), nil
 }
 
+
+type StructLogRes struct {
```

Comment:
Please add a doc to all newly introduced types and functions. Also is there a reason for making these encoding fields public? Private should be fine imho.